cmd/runner.go

```diff
@@ -67,8 +66,7 @@ func addStartCmd(runnerCmd *cobra.Command) {
 
 	startCmd.Flags().StringVar(&host, "host", "0.0.0.0", "Trento Runner API host")
 	startCmd.Flags().IntVar(&port, "port", 8080, "Trento Runner API port")
-	startCmd.Flags().StringVar(&apiHost, "api-host", "0.0.0.0", "Trento web server API host")
-	startCmd.Flags().IntVar(&apiPort, "api-port", 8080, "Trento web server API port")
+	startCmd.Flags().StringVar(&callbacksUrl, "callbacks-url", "http://0.0.0.0/api/runner/callbacks", "Trento web server runner callbacks API url")
```

Comment:
Mayhe

Reply:
Ok, you are right! I will remove the default value

Reply:
Done in: https://github.com/trento-project/runner/pull/8/commits/5db922e242388155c69821ba2b3691d1f90cfd1f